scripts/tests/run.mjs

```diff
@@ -128,18 +141,12 @@ if (snapshots) {
   process.argv.push('--update-snapshots');
 }
 
-// Handle runner stopping with correct exit code
-let runner = undefined;
-const stopRunner = () => {
-  if (runner) {
-    runner.stop();
-    process.exit(0);
-  } else {
-    process.exit(1);
+process.on('uncaughtException', (error) => {
```

Comment:
looks like testing code. Don't forget to clean them up.

Reply:
Removed